### DISCUSSION TITLE
Tested and works with mini

### DISCUSSION THREAD

**Discussion by imbrianj** [vwt12eh8/hassio-ecoflow#31] [General]
I have a Delta Mini as well as a Delta Pro. All seem to be working well with this integration.

**Comment by vwt12eh8** [comment]
Thank you very much. I will update the documentation.

**Comment by vwt12eh8** [comment]
fixed https://github.com/vwt12eh8/hassio-ecoflow/commit/ec2213ee27fc7a4f1c2f4db79f83d860dc33ab2c
